rita_client/src/heartbeat/mod.rs

```diff
@@ -92,9 +136,10 @@ pub fn send_udp_heartbeat() {
     }
 
     trace!("attempting to send heartbeat");
-    let dns_request = Resolver::from_registry()
-        .send(resolver::Resolve::host(heartbeat_url.to_string()))
-        .timeout(CLIENT_LOOP_TIMEOUT);
+    // let dns_request = Resolver::from_registry()
+    //     .send(resolver::Resolve::host(heartbeat_url.to_string()))
+    //     .timeout(CLIENT_LOOP_TIMEOUT);
+    let _dns_request = heartbeat_url.to_socket_addrs();
```

Comment:
this looks like a throwaway assignment?

Reply:
changed from a throwaway to be used in _res further down